deno/gateway/v10.ts

```diff
@@ -707,7 +707,10 @@ export type GatewayChannelModifyDispatch = DataPayload<
  * https://discord.com/developers/docs/topics/gateway-events#channel-update
  * https://discord.com/developers/docs/topics/gateway-events#channel-delete
  */
-export type GatewayChannelModifyDispatchData = APIChannel;
+export type GatewayChannelModifyDispatchData = APIChannel & {
+	type: Exclude<GuildChannelType, ThreadChannelType>;
+	guild_id: Snowflake;
+};
```

Comment:
Does this actually handle the type correctly? Making it all ConcreteType + guild_id?